python/ccxt/fcoin.py

```diff
@@ -415,6 +415,7 @@ def parse_order(self, order, market=None):
             'filled': filled,
             'average': None,
             'status': status,
+            'source':self.safe_string(order, 'source'),
```

Comment:
I'd close this PR for two reasons:
1. It's not in line with the contributing guidelines, as py/php is transpiled from JS, so we should add changes to the source js file, as described here:

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-contribute-code
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#multilanguage-support
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

2. Because the `source` field does not conform to the order spec from the Manual: https://github.com/ccxt/ccxt/wiki/Manual#order-structure , we prefer to keep it in the `info` section of the response, so you can still access it within the order.